src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -226,11 +233,62 @@ class PeerActor(
     context become waitingForConnectionResult(newConnection, noRetries)
   }
 
+  def handleEvmMptFastDownload(rlpxConnection: RLPxConnection): Receive = {
```

Comment:
@AlanVerbner / @LukasGasior1 adding another actor will increase communication overhead and potentially will make application harder to debug, maybe we can just separate logic to different class like here:https://github.com/input-output-hk/etc-client/compare/feature/peerFastDownloadMpt...feature/peerFastDownloadMpt_separateUpload